setup.py

```diff
@@ -21,9 +21,9 @@ def find_stub_files(name: str) -> List[str]:
     readme = f.read()
 
 dependencies = [
-    "mypy>=0.782,<0.790",
-    "typing-extensions",
-    "django",
+    'mypy>=0.782,<=0.790',
+    'typing-extensions',
+    'django',
```

Comment:
Btw, let's use `black` on `setup.py` as well.

Reply:
@kszmigiel and @sobolevn sorry for commenting on merged PR, does this means that it supports any new version of mypy, now mypy has updated version to `0.812` in the table in Readme file:https://github.com/typeddjango/django-stubs#version-compatibility
django-stubs supports till mypy version 0.790
thank you